example/src/streams.ts

```diff
@@ -0,0 +1,117 @@
+
+// Load e2e environment variables into process.env
+require('dotenv').config({ path: 'example.env' });

```

Comment:
Is there an updated example.env? Should that include the constants for E2E_TEST_...?

Reply:
Updated example.env and migrated ids to this.